packages/action-link-test/testUpdateMessage.js

```diff
@@ -0,0 +1,10 @@
+RocketChat.callbacks.add('afterSaveMessage', function(message, room) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['room' is defined but never used.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2674162886/issues/source?bid=3199634&fileBranchId=3333125#l1)